src/System.Windows.Forms.Primitives/src/System/Windows/Forms/Internals/ClientUtils.cs

```diff
@@ -112,7 +112,7 @@ public SequentialEnumInfo(Type t)
             public int MaxValue;
         }
 
-        private static void Debug_SequentialEnumIsDefinedCheck(Enum value, int minVal, int maxVal)
+        private static void Debug_SequentialEnumIsDefinedCheck<TEnum>(TEnum value, int minVal, int maxVal) where TEnum : struct, System.Enum
         {
             Type t = value.GetType();
```

Comment:
You could probably replace GetType() with typeof() because value is guaranteed to be an enum and an enum is "sealed".
```suggestion
            Type t = typeof(TEnum);
```